### PR TITLE
Update pipeline

### PR DESCRIPTION
**Pull request by sowu880**
Add condition for notice.

### CODE REVIEW

File: release.yml [hunk 1]
```diff
@@ -9,8 +9,8 @@ variables:
   testProjects: "**/*Tests/*.csproj"
   publishProject: "**/*Tool/*.csproj"
   buildConfiguration: 'Release'
-  major: 2
-  minor: 2
+  major: 3
```
Comment (BoyaWu10) [inline release.yml:12]: Just want to double that are we going to update the major version here?

Comment (BoyaWu10) [reply release.yml:12]: Just omit this comment. I find that we already have this version in releases.

**Review by BoyaWu10 — APPROVED** [review]

**Comment by BoyaWu10** [comment]
Same changes merged via PR #144.

**Comment by sowu880** [comment]
Update in pr #144

**Pull request closed without merging**